Autor.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
```suggestion
<?php

```
Agregar una línea debajo de `<?php`